CMakeLists.txt

```diff
@@ -3,10 +3,12 @@ project(rtrlib)
 cmake_minimum_required(VERSION 2.6)
 
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)
-
+set(NOSSH false)
```

Comment:
I'm confused, this does not address my last concern in #47. Suppose that for some reason libssh is not found. Then RTRlib will not be compiled with SSH support. Hence, "cmake -D CMAKE_BUILD_TYPE=NoSSH ." as well as "cmake -D CMAKE_BUILD_TYPE=Release ." will lead to the same outcome. Right?